src/examples/js/ProductClientExample.js

```diff
@@ -0,0 +1,24 @@
+import { Zecko } from 'zecko-js';
+
+class ProductClientExample {
```

Comment:
We need to add more examples here and in collection client based on pagination params. Let's discuss